### PR TITLE
fix build error

### PR DESCRIPTION
**Pull request by baentsch**
This fixes the nightly build errors by adding `picnic3` instead of `picnic2`. 

OIDs have been counted up.

##### Checklist
- [x] documentation is added or updated
- [x] tests are added or updated (automatic)
CCI execution [here](https://app.circleci.com/pipelines/github/baentsch/openssl/90/workflows/0aaeef2c-e5a7-436c-a1b5-561173858783)

### CODE REVIEW

File: crypto/objects/objects.txt [hunk 1]
```diff
@@ -1782,9 +1782,9 @@ dstu4145le 2 9 : uacurve9 : DSTU curve 9
 1 3 6 1 4 1 311 89 2 1 1 : picnicl1fs : picnicl1fs
 1 3 6 1 4 1 311 89 2 1 2 : p256_picnicl1fs : p256_picnicl1fs
 1 3 6 1 4 1 311 89 2 1 3 : rsa3072_picnicl1fs : rsa3072_picnicl1fs
-1 3 6 1 4 1 311 89 2 1 11 : picnic2l1fs : picnic2l1fs
-1 3 6 1 4 1 311 89 2 1 12 : p256_picnic2l1fs : p256_picnic2l1fs
-1 3 6 1 4 1 311 89 2 1 13 : rsa3072_picnic2l1fs : rsa3072_picnic2l1fs
+1 3 6 1 4 1 311 89 2 1 18 : picnic3l1 : picnic3l1
```
Comment (christianpaquin) [inline crypto/objects/objects.txt]: For the last element of the OID, I had reserved 0x for picnic, 1x for picnic2, and I would put 2x for picnic 3. Can you set the picnic3 OIDs to "1 3 6 1 4 1 311 89 2 1 21", "... 22", and "... 23"?

**Review by christianpaquin — DISMISSED** [review]

File: crypto/objects/objects.txt (hunk 1 above)
Comment (baentsch) [reply crypto/objects/objects.txt]: So done. Please check again.

Comment (baentsch) [reply crypto/objects/objects.txt]: For the avoidance of doubt: I won't merge this but suggest @xvzcf does when `openssl` release is done.

**Review by dstebila — CHANGES_REQUESTED** [review]
Can you rebase this off of master, and then undo #221 and #222 so it is no longer pinned to liboqs 0.3.0?  Once this passes with liboqs HEAD it should be ready to merge.

**Comment by baentsch** [comment]
> Can you rebase this off of master, and then undo #221 and #222 so it is no longer pinned to liboqs 0.3.0? Once this passes with liboqs HEAD it should be ready to merge.

So done. Please re-review and approve for merge.

**Review by dstebila — APPROVED** [review]

**Pull request merged**